quarkus/server/distribution/LICENSE

```diff
@@ -2086,6 +2086,12 @@ License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE
 
 --------------------------------------------------------------------------------
 
+Group: org.eclipse.angus Name: angus-activation Version: 2.0.2
```

Comment:
Just curious: this dependency comes from the Quarkus BOM, but it's not used in Polaris. Do we still need to include it?

Reply:
It appears in the distribution.